tests/test_cases/test_cocotb/Makefile

```diff
@@ -19,7 +19,7 @@ MODULE := "\
 	test_async_generators,\
 	test_handle,\
 	test_logging,\
-	test_pytest,\
+	pytest_assertion_rewriting,\
```

Comment:
Can you add a comment in this file explaining why it is deliberately named in a different way to all the others?

Reply:
Added the comment above, it doesn't quite work on this line.